test/e2e/instrumentation/logging/generic_soak.go

```diff
@@ -58,14 +58,14 @@ var _ = instrumentation.SIGDescribe("Logging soak [Performance] [Slow] [Disrupti
 		var wg sync.WaitGroup
 		wg.Add(loggingSoak.Scale)
 		for i := 0; i < loggingSoak.Scale; i++ {
-			go func() {
+			go func(i interface{}) {
```

Comment:
Could you use more concrete type? e.g. `int`

Reply:
@serathius changes done. Thank you.